src/PhpSpec/Console/Application.php

```diff
@@ -147,6 +147,13 @@ protected function loadConfigurationFile(InputInterface $input, ServiceContainer
 
                     $extension->load($container);
                 }
+            } elseif ('matchers' === $key && is_array($val)) {
+                foreach ($val as $class) {
+                    $id = strtolower(str_replace('Matcher', '', substr($class, strrpos($class, '\\') + 1)));
```

Comment:
My purpose was to avoid silently overriding matchers.

Removing the `Matcher` suffix is only esthetic, and does not serve any real purpose when I look back at it.

In fact, the id could be the class name provided in the configuration (ie: `matchers.App\Matcher\FooMatcher`), I think it should work (and would speed up the registration process a bit).